astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -1243,7 +1244,7 @@ def test_data_out_of_range(parallel, fast_reader, guess):
         fast_reader["parallel"] = parallel
         if fast_reader.get("use_fast_converter"):
             rtol = 1.0e-15
-        elif np.iinfo(np.int_).dtype == np.dtype(np.int32):
```

Comment:
I could imagine that the windows failures are related to this change. I just got the `sys.maxsize` replacement from stackoverflow. But I guess on 32bit one could also test `if np.intp == np.int32`.

Reply:
It is obviously not triggering the `pytest.xfail`, but it is also not failing – just missing 1 expected warning, so maybe that part should instead be changed below.

Reply:
pytest refuses to run locally on Windows for other reasons and I don't have time to fix it. I think to match existing behavior, it is best to just xfail if win32 is detected. I added a commit to do this.